pkg/project/classic/dispatcher/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN yum -y install httpd wget mod_ssl
 
 # Download & install dispatcher module
 RUN mkdir -p /tmp/dispatcher && cd /tmp/dispatcher && \
-    wget https://download.macromedia.com/dispatcher/download/dispatcher-apache2.4-linux-aarch64-${DISPATCHER_VERSION}.tar.gz -O dispatcher.tar.gz && \
+    wget https://download.macromedia.com/dispatcher/download/dispatcher-apache2.4-linux-$(uname -m)-${DISPATCHER_VERSION}.tar.gz -O dispatcher.tar.gz && \
```

Comment:
this is crazy - @tomasz-sobczyk-wttech take a look ;)

on Windows we had here x86_64; on M1 Mac aarch;

Docker is not isolating arch ;)